web/src/components/RunStatusIcon/RunStatusIcon.tsx

```diff
@@ -0,0 +1,38 @@
+import {Tooltip} from 'antd';
+import TestRun, {isRunStateFailed, isRunStateFinished, isRunStateStopped} from 'models/TestRun.model';
+import RequiredGatesResult from 'models/RequiredGatesResult.model';
+import * as S from './RunStatusIcon.styled';
+import RunTooltipTitle from './RunTooltipTitle';
+
+interface IProps {
+  state: TestRun['state'];
+  requiredGatesResult: RequiredGatesResult;
+}
+
```

Comment:
This component encapsulates the run status icon that should be used everywhere in the app